### DISCUSSION TITLE
Compiling RPM Version 4.8 on Centos 7

### DISCUSSION THREAD

**Discussion by vibfft** [rpm-software-management/rpm#2895] [General]
Hi,

I would like to compile RPM version 4.8.0.
I downloaded the source for 4.8.X branch, but I am not sure how to compile the rpm version 4.8.0 on centos 7 platform.
I look foward to hearing from you.

-Stephen

**Comment by pmatilai** [comment]
You compile it the same as any autotools software: grab the release tarball (not git branch), ./configure and so on. The rpm version being several years older than the distro you'd be compiling it on could cause issues with library and compiler compatibility and such. Any breakage you get to keep to yourself, that version is over a decade out of support.
